<ui version="4.0" >
 <class>NotesWindow</class>
 <widget class="QMainWindow" name="NotesWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Notes</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>22</y>
     <width>852</width>
     <height>667</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" >
    <property name="spacing" >
     <number>4</number>
    </property>
    <property name="margin" >
     <number>6</number>
    </property>
    <item>
     <widget class="QFrame" name="radarSettings_" >
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>4</number>
       </property>
       <item>
        <widget class="QCheckBox" name="radarTransmitting_" >
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip" >
          <string>Checked if radar is transmitting</string>
         </property>
         <property name="statusTip" >
          <string/>
         </property>
         <property name="text" >
          <string>Transmitting</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="radarFrequency_" >
         <property name="focusPolicy" >
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="toolTip" >
          <string>Radar transmitter frequency</string>
         </property>
         <property name="statusTip" >
          <string/>
         </property>
         <property name="suffix" >
          <string> MHz</string>
         </property>
         <property name="minimum" >
          <double>1.000000000000000</double>
         </property>
         <property name="value" >
          <double>24.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>16</width>
           <height>27</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="radarRotating_" >
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip" >
          <string>Checked if radar is rotating</string>
         </property>
         <property name="text" >
          <string>Rotating</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="radarRotationRate_" >
         <property name="focusPolicy" >
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="toolTip" >
          <string>Rate of radar rotation</string>
         </property>
         <property name="suffix" >
          <string> rpm</string>
         </property>
         <property name="minimum" >
          <double>0.010000000000000</double>
         </property>
         <property name="value" >
          <double>6.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>16</width>
           <height>27</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="drfmOn_" >
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip" >
          <string>Checked if DRFM is in use</string>
         </property>
         <property name="text" >
          <string>DRFM</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="drfmConfigLabel_" >
         <property name="text" >
          <string>Config:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="drfmConfig_" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy" >
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="toolTip" >
          <string>ActiveDRFM configuration file</string>
         </property>
         <property name="maxLength" >
          <number>32</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="recordingInfo_" >
      <property name="frameShape" >
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>0</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_4" >
         <property name="font" >
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>Recording Status</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="recordingInfo2_" >
         <property name="frameShape" >
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="leftMargin" >
           <number>4</number>
          </property>
          <property name="topMargin" >
           <number>0</number>
          </property>
          <property name="rightMargin" >
           <number>4</number>
          </property>
          <property name="bottomMargin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Now:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="now_" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>HH:MM:SS</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="insertNow_" >
            <property name="text" >
             <string>Insert</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>20</width>
              <height>32</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>Elapsed:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="elapsed_" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>HH:MM:SS</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="insertElapsed_" >
            <property name="text" >
             <string>Insert</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>40</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="recordingStop_" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="palette" >
             <palette>
              <active>
               <colorrole role="ButtonText" >
                <brush brushstyle="SolidPattern" >
                 <color alpha="255" >
                  <red>255</red>
                  <green>51</green>
                  <blue>51</blue>
                 </color>
                </brush>
               </colorrole>
              </active>
              <inactive>
               <colorrole role="ButtonText" >
                <brush brushstyle="SolidPattern" >
                 <color alpha="255" >
                  <red>255</red>
                  <green>51</green>
                  <blue>51</blue>
                 </color>
                </brush>
               </colorrole>
              </inactive>
              <disabled>
               <colorrole role="ButtonText" >
                <brush brushstyle="SolidPattern" >
                 <color alpha="255" >
                  <red>106</red>
                  <green>104</green>
                  <blue>100</blue>
                 </color>
                </brush>
               </colorrole>
              </disabled>
             </palette>
            </property>
            <property name="focusPolicy" >
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="toolTip" >
             <string>Stop the recording process</string>
            </property>
            <property name="text" >
             <string>Stop Recording</string>
            </property>
            <property name="shortcut" >
             <string>Ctrl+S</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="SideCar::GUI::Master::NotesEdit" name="notes_" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="elapsed_1" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Drops:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="drops_" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="elapsed_3" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Duplicates:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="duplicates_" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="close_" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip" >
         <string>Close window</string>
        </property>
        <property name="text" >
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit" >
    <property name="title" >
     <string>Edit</string>
    </property>
    <addaction name="actionUndo" />
    <addaction name="actionRedo" />
    <addaction name="separator" />
    <addaction name="actionCut" />
    <addaction name="actionCopy" />
    <addaction name="actionPaste" />
    <addaction name="separator" />
    <addaction name="actionInsert_Time" />
    <addaction name="actionInsert_Elapsed" />
    <addaction name="separator" />
    <addaction name="actionStop_Recording" />
   </widget>
   <addaction name="menuEdit" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>689</y>
     <width>852</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <action name="actionUndo" >
   <property name="text" >
    <string>Undo</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo" >
   <property name="text" >
    <string>Redo</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionCopy" >
   <property name="text" >
    <string>Copy</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCut" >
   <property name="text" >
    <string>Cut</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste" >
   <property name="text" >
    <string>Paste</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionInsert_Time" >
   <property name="text" >
    <string>Insert Time</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionStop_Recording" >
   <property name="text" >
    <string>Stop Recording</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+.</string>
   </property>
  </action>
  <action name="actionInsert_Elapsed" >
   <property name="text" >
    <string>Insert Elapsed</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+E</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SideCar::GUI::Master::NotesEdit</class>
   <extends>QTextEdit</extends>
   <header>NotesEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>close_</sender>
   <signal>clicked()</signal>
   <receiver>NotesWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>123</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel" >
     <x>85</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
